tests/vision_test.cpp

```diff
@@ -30,6 +30,7 @@ void full_map_test( const std::vector<std::string> &setup,
     const ter_id t_floor( "t_floor" );
     const ter_id t_utility_light( "t_utility_light" );
     const efftype_id effect_narcosis( "narcosis" );
+    const ter_id t_flat_roof( "t_flat_roof" );
 
     g->place_player( tripoint( 60, 60, 0 ) );
     g->u.worn.clear(); // Remove any light-emitting clothing
```

Comment:
Something like this might do it.
```suggestion
    g->u.worn.clear(); // Remove any light-emitting clothing
    const int mapsize = g->m.getmapsize() * SEEX;
    for( int z = 1; z <=10; ++z ) {
        for( int x = 0; x < mapsize; ++x ) {
            for( int y = 0; y < mapsize; ++y ) {
                g->m.set( { x, y, z }, t_open_air, f_null );
            }
        }
    }
```

Reply:
Done that, except in `wipe_map_terrain` rather than here so all tests benefit.

Reply:
Yea just an example.
The *right* way to make this fast is to inject some test mapgen that gets us to a sane state by using COW as outlined in https://github.com/CleverRaven/Cataclysm-DDA/issues/29674